### PR TITLE
Add RTCP parsing

### PR DESCRIPTION
**Pull request by maxhawkins**
Just two more to go after this...

### CODE REVIEW

File: pkg/rtcp/reception_report.go [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+	// sequence number cycles.
+	LastSeq uint32
+	// An estimate of the statistical variance of the RTP data packet
+	// interarrival time, measured in timestamp units and expressed as an
+	// unsigned integer.
+	Jitter uint32
+	// The middle 32 bits out of 64 in the NTP timestamp received as part of
+	// the most recent RTCP sender report (SR) packet from source SSRC. If no
+	// SR has been received yet, the field is set to zero.
+	LastSR uint32
```
Comment (Sean-Der) [inline pkg/rtcp/reception_report.go]: Can we do full for public names `LastSenderReport` and `LastSequenceNumber`

File: pkg/rtcp/source_description.go [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+
+import (
+	"encoding/binary"
+
+	"github.com/pkg/errors"
+)
+
+// RTP SDES item types registered with IANA. See: https://www.iana.org/assignments/rtp-parameters/rtp-parameters.xhtml#rtp-parameters-5
+const (
+	SDESEnd      = iota // end of SDES list                RFC 3550, 6.5
```
Comment (Sean-Der) [inline pkg/rtcp/source_description.go]: Can we make these an enum (and have a short ToString() method)

File: pkg/rtcp/reception_report.go (hunk 1 above)
Comment (maxhawkins) [reply pkg/rtcp/reception_report.go]: Yep

File: pkg/rtcp/source_description.go (hunk 2 above)
Comment (maxhawkins) [reply pkg/rtcp/source_description.go]: OK

Comment (maxhawkins) [reply pkg/rtcp/source_description.go]: I'll do the same for the RTCP packet types

**Comment by maxhawkins** [comment]
I have a sketch of a `srtp.Reader` class that I'll upload tonight.

I need to do some refactoring first because the BYE packet needs access to the header to parse correctly.

Thanks for taking a look.

**Comment by maxhawkins** [comment]
I'm going to post in the issue (#119) about RTCPeerConnection options.

**Comment by Sean-Der** [comment]
Closing in favor of #160

**Pull request closed without merging**